receiver/flinkmetricsreceiver/metadata.yaml

```diff
@@ -0,0 +1,328 @@
+name: flinkmetricsreceiver
+
+resource_attributes:
+  # These resource attributes are Flinks system scope variables, which contains context information about metrics. These are required to uniquely identify incoming metrics as the same job can run multiple times concurrently. See https://nightlies.apache.org/flink/flink-docs-release-1.14/docs/ops/metrics/#system-scope for more information.
+  host.name:
+    description: The host name.
+    type: string
+  flink.taskmanager.id:
+    description: The taskmanager ID.
+    type: string
+  flink.job.name:
+    description: The job name.
+    type: string
+  flink.task.name:
+    description: The task name.
+    type: string
+  flink.subtask.index:
+    description: The subtask index.
+    type: string
+  flink.resource.type:
+    description: The flink scope type in which a metric belongs to.
+    type: string
+    enum: [ jobmanager, taskmanager ]
+
+attributes:
+  operator_name:
+    description: The operator name.
+    type: string
+  garbage_collector_name:
+    description: The names for the parallel scavenge and garbage first garbage collectors.
+    type: string
+    enum: [ PS_MarkSweep, PS_Scavenge, G1_Young_Generation, G1_Old_Generation ]
+  checkpoint:
+    description: The number of checkpoints completed or that failed. 
+    type: string
+    enum: [ completed, failed ]
+  record:
+    description: The number of records received in, sent out or dropped due to arriving late.
+    type: string
+    enum: [ in, out, dropped ]
+
+metrics:
+  flink.jvm.cpu.load:
+    enabled: true
+    description: The CPU usage (%) of the JVM for a jobmanager or taskmanager.
+    unit: 1
```

Comment:
%?

Reply:
removed

Reply:
I'm confused. I thought you said this metric is measured in %. That's my interpretation from the article you linked.

Reply:
@JonathanWamsley PTAL

Reply:
Ok, should the units be `%` or not?

Reply:
Yes they should be. The unit can be 1 but not `%` because that will break in the metadata generation. Maybe `"{percentage}"` instead?

Reply:
`"%"` should work

Reply:
updated unit